msmbuilder/decomposition/tica.py

```
@@ -321,7 +325,12 @@ def transform(self, sequences):
             X_transformed = np.dot(X, self.components_.T)
 
             if self.weighted_transform:
+                warnings.warn("weighted_transform is deprecated. "
+                              "Please use kinetic_mapping",
+                              DeprecationWarning)
                 X_transformed *= self.timescales_
+            elif self.kinetic_mapping:
```

If both are set to `True`, can you raise an error?